storage_service/locations/models/package.py

```diff
@@ -1183,12 +1192,12 @@ def create_new_pointer_file_with_replication(
         replication_event = self.create_replication_event(
             replica_package, event_uuid=replication_event_uuid, agents=ss_agents
         )
-        old_premis_events.append(replication_event)
+        old_premis_events += (replication_event,)
```

Comment:
I suspect I'm missing something obvious but I'm not sure why this change is needed for this? `old_premis_events` I believe to be a list at this stage so adding a one item tuple, or appending to the list has the same effect? Same q. with the other `+= (tuple, )` lines below.

Reply:
This is a good point — I was getting a "tuple has no attribute append" here, but looking at the metsrw code again, I don't see it returning a tuple. I'll have another look.

Reply:
I think I got the lines mixed up here: `new_relationships` is a tuple, but `old_premis_events` and `old_premis_agents` are lists. `+=` also works for lists, but I'll switch back to `append`.